### ISSUE TITLE
White spaces in the pax runner root path causes pax runner to not find provisioning folder [PAXRUNNER-287]

### ISSUE THREAD

**Issue by ops4j-issues** [ops4j/org.ops4j.pax.runner#310]
**[M](https://ops4j1.jira.com/secure/ViewProfile.jspa?accountId=557058:c4c7d61b-34fa-4e34-961a-dab256a3cc6f)** created **[PAXRUNNER-287](https://ops4j1.jira.com/browse/PAXRUNNER-287)** 

I have the following configuration:

```java
C:\some folder with spaces\pax-runner
C:\some folder with spaces\pax-runner\bin\pax-run.bat
C:\some folder with spaces\pax-runner\bin\runner.args
...
C:\some folder with spaces\pax-runner\plugins\someBundle.jar
...
```

The runner.args looks like this:

```java
# runner.args
--p=e 
--version=3.5.0 

../plugins 
```

When I now try to start par runner, it can't find the plugins folder. As soon as I copy it to a folder without white spaces, everything works fine.

```java
 -> Scan bundles from [../plugins]
 -> Scan bundles from [scan-dir:file:/C:/some folder with spaces/pax-runner/bin/../plugins/]
         ___
        /  /
       /  / Oops, there has been a problem!
      /  /
     /__/   org.ops4j.pax.runner.provision.MalformedSpecificationException: Specification [file:/C:/some folder with spaces/pax-runner/bin/../plugins/] could not be used
    ___
   /__/     Use --log=debug to see details.
```

---

**Affects:** 1.1.1
**Fixed in:** 1.2.0
Votes: 0, Watches: 0

---
**Referenced issues**

**depends on:**
* ops4j/org.ops4j.pax.scanner#7 - Scanning a directory with a name containing spaces does not work [PAXSCANNER-6]

**Comment by ops4j-issues** [comment]
**[Alin Dreghiciu](https://ops4j1.jira.com/secure/ViewProfile.jspa?accountId=557058:a786db11-2013-41fb-af1d-ca5eb8f74aa0)** commented

This is actually an Pax Scanner issue now.

**Comment by ops4j-issues** [comment]
**[Alin Dreghiciu](https://ops4j1.jira.com/secure/ViewProfile.jspa?accountId=557058:a786db11-2013-41fb-af1d-ca5eb8f74aa0)** commented

Released as part of 1.2.0
